data/json/npcs/hints.json

```diff
@@ -90,6 +90,37 @@
         "In a lot of places, there's little hallways connecting the subways with the sewers, with heavy metal doors on both ends.  It's a perfect place to sleep!",
         "Be careful of drinking water from rivers and stuff, it's a good way to get sick.  But if you have a water purifier, it'll make it safe.",
-        "Autodocs are probably your best bet to install bionics - if you can find one!  However, I heard they won't work if you don't bring anesthetics to put you in a sleep."
+        "Autodocs are probably your best bet to install bionics - if you can find one!  However, I heard they won't work if you don't bring anesthetics to put you in a sleep.",
+        "Be wary of anything deeper than your average basement. Some folks heard scary screams from the ventilation shafts of mines and labs. Very unsettling screams.",
+        "A screwdriver a day keeps the scurvy away!",
+        "Hungrier, than usual? Natural oils can help. Not tasty at all, but who cares when eating your leg is the second option?",
+        "Terrain can turn the tide of a battle, make sure you use it against your enemies, lest it be used against you.",
+        "Folks that passed by the mine said something about foul smell. If you plan a visit there consider taking a a gas mask with you.",
+        "Knowledge is power. Seriously, just pick up a book.",
+        "Nothing can kill you if everything is already dead. Well, except cold, hunger, and… never mind.",
+        "I met a girl that insisted that she saw a land shark boring throught rock, walls and dirt alike. I'd consider it a fable, but I've seen things, and can't just say it's a lie.",
+        "Boil first, then drink. Ten men caravan once drank from a spring, and they are now known as a three man caravan.",
+        "I've once seen a full duffel bag pass hands five times in a week. Having a lot of stuff is a moot point if you can’t run from zombies.",
+        "Tim says you can drive a car through a horde of zombies. Tim is now a zombie. Any questions?",
+        "They said: go solar, save the environment and yourself. Well... there is no environment to save now, but one can still save thyself I guess.",
+        "If you can’t find a knife try smashing potted plants. It just might give you everything you need to make one.",
+        "What's the difference between a good and a bad choke point. The good one has another back door behind you.",
+        "So, methinks: if you could convince the cop-bots that you are their superior...",
+        "You’d be surprised how many items can be disassembled into their components. This guy around here, McSomething whatever his name is, is a master at this.",
+        "A soldering iron can be an aspiring mechanics’ best friend. It can also cauterize a wound with it, but as many peple died as lived from that treatment, so I guess it's a last resort.",
```

Comment:
```suggestion
        "A soldering iron can be an aspiring mechanics’ best friend.  It can also cauterize a wound with it, but as many people died as lived from that treatment, so I guess it's a last resort.",
```